core/extension/extension_api_dump.cpp

```diff
@@ -1020,7 +1020,8 @@ Dictionary GDExtensionAPIDump::generate_extension_api(bool p_include_docs) {
 						d2["is_required"] = (F.flags & METHOD_FLAG_VIRTUAL_REQUIRED) ? true : false;
 						d2["is_vararg"] = false;
 						d2["is_virtual"] = true;
-						// virtual functions have no hash since no MethodBind is involved
+						d2["hash"] = mi.get_hash();
```

Comment:
Can you imagine that there's any way how the hash is going to change in the future?

Just because I remember the big issue we had with re-hashing methods in 4.0.x, and also a few other cases where it wasn't clear how const-ness and some other properties would affect hashes.